src/pipeline/pipeline.go

```diff
@@ -102,7 +102,8 @@ func (p *pipeline) input(name string) error {
 func (p *pipeline) inject(name string) error {
 	switch {
 	case strings.HasPrefix(name, "s3://"):
-		bucket, err := s3utils.NewBucket(name)
+		bucket, err := s3utils.GetBucket(name)
+		client := s3.New(nil)
```

Comment:
Hum actually, I'm going to leave it like that. No point of making a function out of this.